README.rst

```diff
@@ -1,76 +1,51 @@
-|Build Status| |Codecov| |license|
+|Build Status|
 
-frontend-template-application
-=================================
+frontend-app-skills
+==============================
 
-Please tag **@edx/fedx-team** on any PRs or issues.  Thanks.
+Please tag **@openedx/2u-aperture** on any PRs or issues.
```

Comment:
We're not going to want to reference the `openedx` GH org team, that doesn't exist in the `edx` GH org. I believe the team should be `edx/edx-aperture`.